src/models/response/AuthResponse.ts

```diff
@@ -0,0 +1,7 @@
+import { IUser } from "../request/IUser";
+
+export interface AuthResponse {
+  accessToken: string;
+  refreshToken: string;
+  user: IUser;
+}
```

Comment:
Also move types to different folders
All types for Auth to Auth folder and so on